tests/HasPermissionsTest.php

```diff
@@ -188,19 +188,15 @@ public function it_can_determine_that_the_user_does_not_have_a_permission()
     }
 
     /** @test */
-    public function it_throws_an_exception_when_the_permission_does_not_exist()
+    public function it_returns_false_when_the_permission_does_not_exist()
```

Comment:
@AlexVanderbist are you okay with this no longer throwing an exception if permission does not exist? 
Is this sort of change too significant to not be considered a breaking change in Spatie terms?

Reply:
Hi! As discussed over email; I definitely agree on not throwing the exception. It's been bugging me for a while but I wasn't sure about removing it as that would probably be a breaking change. 

I'm not sure if this would even be considered a feature. I can't imagine anyone checking if a permission exists based on that exception being thrown... If it's not a feature we can tag it as a patch.